src/Section2/style2.js

```diff
@@ -15,16 +24,28 @@ export const SubTitle = styled.h2`
   font-size: 50px;
   color: #3f3d56;
   padding-bottom: 35px;
+
+  @media (max-width: 768px) {
+    font-size: 45px;
+    color: #3f3d56;
+    padding-bottom: 32px;
+  }
+
+  @media (max-width: 376px) {
+    font-size: 30px;
+    color: #3f3d56;
+    padding-bottom: 35px;
+  }
 `;
 
 export const Block0 = styled.div`
```

Comment:
Same thing about the naming here. Could be something like `SkillsContainer`
Something that makes sense in the context you are in. So if you look your JSX file you can just read the components and understand what they are